### PR TITLE
docs: add info about docker integration

### PR DESCRIPTION
**Pull request by davideicardi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Docs
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by micalevisk** [comment]
what about add a link to this basic repo github template: https://github.com/jmcdo29/nest-docker-template

File: content/fundamentals/container-integration.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+### Container integration
+
+#### Dockerfile template
+
+Here a `Dockerfile` template that you can use:
```
Comment (johannesschobel) [inline content/fundamentals/container-integration.md:5]:
```suggestion
Here is a `Dockerfile` template that can be used as a template:
```

**Review by johannesschobel — CHANGES_REQUESTED** [review]

**Comment by micalevisk** [comment]
I guess this won't get merged due to this: https://github.com/nestjs/nest/pull/5810#issuecomment-740515157

**Comment by johannesschobel** [comment]
> I guess this won't get merged due to this: https://github.com/nestjs/nest/pull/5810#issuecomment-740515157 

Dear @micalevisk , I still think this may be a valuable contribution to the docs to give an idea how to start.. maybe @davideicardi can better highlight that dockerizing an app is highly subjective and depend on various factors (i.e., runtime environment, node engine, ...) and that the snippets provided is just a basic idea..

**Comment by kamilmysliwiec** [comment]
https://github.com/nestjs/nest/pull/5810#issuecomment-740515157

**Pull request closed without merging**